src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -514,6 +514,11 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditor
 
 		const showCellStatusBar = this.configurationService.getValue<boolean>(ShowCellStatusBarKey);
 		this._overlayContainer.classList.toggle('cell-statusbar-hidden', !showCellStatusBar);
+
+		const cellToolbarInteraction = this.configurationService.getValue<string>(CellToolbarVisibility);
+		this._overlayContainer.classList.remove('cell-toolbar-hover');
+		this._overlayContainer.classList.remove('cell-toolbar-click');
+		this._overlayContainer.classList.toggle(`cell-toolbar-${cellToolbarInteraction}`);
```

Comment:
Check that this setting is one of the expected values before adding it as a class, then use `classList.add` instead of toggle here

Reply:
Great catch! Updated the logic here.